model-optimizer/mo/front/extractor.py

```diff
@@ -132,13 +132,13 @@ def attr_getter(node: Node, name: str):
 
 
 def bool_to_str(node: Node, attr: str):
-    # Function converts 0/1 or bool False/True values to str 'false'/'true' which need to appear in IR
+    # Function converts 0/1 or bool False/True or '0'/'1' values to str 'false'/'true' which need to appear in IR
     attribute_name = node.soft_get(attr, None)
     if attribute_name is None:
         return None
     if isinstance(attribute_name, bool):
         return str(attribute_name).lower()
-    elif attribute_name in [0, 1]:
+    elif attribute_name in [0, 1, '0', '1']:
```

Comment:
I recommend to change the Caffe extractor to extract values as boolean attributes and not strings. So the change is file is not needed.

Reply:
Extractor change is more risky. It can't be changed only for MVN, but for all bool attributes in Caffe models. We can have somewhere code that assumes to get string (I don't know such places but can't be sure that we haven't it somewhere).